rules-topics/18.00_glossary.md

```diff
@@ -1,84 +1,201 @@
-|:---|:---:|:---:|:---:|:---:|---|
-| One-Handed Weapon |  3  |  3  |  5  |  8  |-|
-| Two-Hand Weapon|  3  |  3  | 5  | 8 |-|
-| Ranged Weapon  |  3  | 5  |  3  | 8  |-|
-| Shield|  3  |  3  | 5  | 8  |-|
-| Dual Wield |  3  |  3  |  5  | 8  | One-Handed Weapon |
-| Parrying Dagger |  2  | 2  |  2  | 2  |-|
- 
- ### Pool Skills
-| Level | Agility | Stamina |
-|---|---|---|
-| 1 | Backstab | Blind<br>Critical Strike<br>Disarm<br>Slow |
-| 2 | Entangling Shot<br>Poison Shot | Stop Thrust |
-| 3 | Shake It Off | Rapid Refit |
-| 4 | Kneeling Shot<br>Resourceful | Disable Limb<br>Kneel<br>Parry<br>Spell Parry |
-| 5 | Breaching Shot<br>Warbow Training | Critical Silence<br>Vital Blow |
-| 6 | Blinding Shot<br>Waylay | Fear<br>Intercept | 
-| 7 | Ballista<br>Cursing Shot | Critical Curse<br>Destroy |
-| 8 | Surprising Proficiency | Endurance<br>Stun |
-| 9 | Dodge | Regenerate |
-| 10 | Agile Proficiency<br>Murder | Slay<br>Weapon Proficiency |
-
-### Spells
-| Tier | Universal | Arcane | Spirit | Nature |
-|---|---|---|---|---|
-| 1  | Glowing Light| Blinding Light<br>Mana Bolt | Control: Minor Undead<br>Transfer Malady<br>Heal Wounds | Command: Disarm<br>Grasping Grasses<br>Purify Food and Drink |
-| 2  | | Disrupt Aether<br>Stop<br>Mana Bolt| Harm Undead<br>Heal Wounds<br>Healing Touch<br>Preserve Corpse<br>  | Boon of Nature<br>Entangle Foot<br>Plant Spike |
-| 3  | | Limited Time Stop<br>Mana Bolt| Cause Weakness<br>Heal Wounds  | Boon of the Armadillo<br>Boon of the Tiger<br>Freedom  |
-| 4  | Barrier | Conjure Weapon<br>Mana Bolt<br>Temporal Shift†| Heal Wounds<br>Rot Limb  | Entangle Arms<br>Command: Kneel |
-| 5  | Entrap Spirit | Conjure Shield<br>Enchanted Blade<br>Mana Bolt| Cleanse<br>Field Resurrection<br>Heal Wounds<br>Revivify  | <br>Command: Silence<br>Strengthen Resolve<br>Treehome† |
-| 6  | | Limited Teleport†<br>Mana Bolt<br>Nullify Spell<br>Stasis† | Dull Pain<br>Fear<br>Heal Wounds<br>Solidify  | Command: Sleep<br>Entangle Body |
-| 7  | | Destroy<br>Haste<br>Mana Bolt<br>Rift Lock| Curse<br>Heal Wounds<br>Vampiric Touch | Command: Confuse<br>Confine |
-| 8  | Reflect | Bag of Holding<br>Mana Bolt| Destroy Undead<br>Heal Wounds<br>Limited Phylactery†<br>Repair Undead | Boon of the Cat<br>Command: Stun  |
-| 9  | Dispel Magic | Circle of Power<br>Mana Bolt| Create Undead<br>Enervate<br>Heal Wounds<br>  | Treewalk†<br>Regenerate<br>Wrath of Nature, Other |
-| 10 |  | Arcane Torrent<br>Doom| Storm of Curses<br>Life | Sanctuary<br>Spider Web |
+
+| Level | Abilities |
+|---|---|
+| 1 | Critical Backstab |
+| 1 | Blinding Strike |
+| 1 | Slowing Strike |
+| 2 | Disarming Strike |
+| 2 | Entangling Strike |
+| 3 | Kneeling Strike |
+| 3 | Resourceful |
+| 4 | Parry |
+| 4 | Spell Parry |
+| 4 | Weakening Strike |
+| 5 | Breaching Strike |
+| 5 | Silencing Strike |
+| 6 | Fearful Strike |
+| 6 | Waylay |
+| 7 | Critical Empowerment |
+| 7 | Warbow Training |
+| 8 | Stunning Strike  |
+| 8 | Trickle Blade |
```

Comment:
Surprising Proficiency is missing here.

Reply:
Only needs to be added to the agility json